js/src/post.js

```diff
@@ -22,19 +22,18 @@ jQuery(
  */
 jQuery(
 	function( $ ) {
-		$( document ).on(
-			'DOMNodeInserted',
-			function( e ) {
-				var t = $( e.target );
-
-				// WP inserts the quick edit from
-				if ( 'inline-edit' == t.attr( 'id' ) ) {
-					var post_id = t.prev().attr( 'id' ).replace( "post-", "" );
+		const table = document.getElementById( 'the-list' );
+		const config = { attributes: true, childList: true, subtree: true };
```

Comment:
According to the `MutationObserver::observe()` it seems to me `childList: true` and `subtree: true` are enough to detect the `the-list` child node insertion for the quick edit form. It will allow to avoid to trigger the callback on any other mutations like attributes which isn't necessary here.